confignode/src/main/java/org/apache/iotdb/confignode/persistence/partition/PartitionInfo.java

```diff
@@ -98,6 +105,42 @@ public PartitionInfo() {
     this.deletedRegionSet = Collections.synchronizedSet(new HashSet<>());
   }
 
+  public void addMetrics() {
+    if (MetricConfigDescriptor.getInstance().getMetricConfig().getEnableMetric()) {
+      MetricsService.getInstance()
+          .getMetricManager()
+          .getOrCreateAutoGauge(
+              Metric.PARTITION_TABLE.toString(),
+              MetricLevel.CORE,
+              storageGroupPartitionTables,
+              Map::size,
+              Tag.NAME.toString(),
+              "number");
+      MetricsService.getInstance()
+          .getMetricManager()
+          .getOrCreateAutoGauge(
+              Metric.REGION.toString(),
+              MetricLevel.IMPORTANT,
+              this,
+              o -> o.getTotalRegionCountAndUpdateMetric(TConsensusGroupType.SchemaRegion),
+              Tag.NAME.toString(),
+              "total",
+              Tag.TYPE.toString(),
+              TConsensusGroupType.SchemaRegion.toString());
+      MetricsService.getInstance()
+          .getMetricManager()
+          .getOrCreateAutoGauge(
+              Metric.REGION.toString(),
```

Comment:
Metric.DATAREGION?

Reply:
I use type tag to filter which is more convenience.